routers.js

```diff
@@ -0,0 +1,104 @@
+const express = require("express");
+
+const db = require("./data/db.js");
+
+const router = express.Router();
+
+router.get("/", async (req, res) => {
+  try {
+    const posts = await db.find();
+    res.status(200).json(posts);
+  } catch (error) {
+    console.log(error);
+    res
+      .status(500)
+      .json({ message: "The posts information could not be retrieved." });
+  }
+});
+
+router.get("/:id", async (req, res) => {
+  const { id } = req.params;
+  try {
+    const post = await db.findById(id);
+    if (post[0]) {
```

Comment:
this could be simplified a little if you use array destructuring when capturing the post like `const [post] = await db.findById(id)`, then you wouldn't have to reference `post[0]` in your conditional, and the post you send back with status 200 would be just the object itself instead of an array containing the object